File: EDMTDev/3_Load_Image_From_URL/app/src/main/java/com/rasyidcode/a3_load_image_from_url/MainActivity.java
package com.rasyidcode.a3_load_image_from_url;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.Toast;

import com.squareup.picasso.Callback;
import com.squareup.picasso.Picasso;

public class MainActivity extends AppCompatActivity {

    public static String IMAGE_URL = "https://foto.wartaekonomi.co.id/files/arsip_foto_2019_07_29/kimi_hime_120005_big.webp";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ImageView imageView = findViewById(R.id.image_test);
        Picasso.get().load(IMAGE_URL)
                .placeholder(R.drawable.ic_launcher_background)
                .error(R.drawable.ic_launcher_background)
                .into(imageView, new Callback() {
                    @Override
                    public void onSuccess() {
                        Toast.makeText(MainActivity.this, "Load image successful", Toast.LENGTH_SHORT).show();
                    }

                    @Override
                    public void onError(Exception e) {
                        Toast.makeText(MainActivity.this, "Load image failed", Toast.LENGTH_SHORT).show();
                    }
                });
    }
}

File: from-article/codelabs-tuts/OnePointTwo_DiceRoller/app/src/main/java/me/jamilalrasyidis/onepointtwo_diceroller/MainActivity.kt
package me.jamilalrasyidis.onepointtwo_diceroller

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnRollDice: Button = findViewById(R.id.btn_roll_dice)
        btnRollDice.setOnClickListener { rollDice() }
    }

    private fun rollDice() {
        val randomInt = Random.nextInt(6) + 1
        val tvResult: TextView = findViewById(R.id.tv_result)
        tvResult.text = randomInt.toString()
    }
}
